CleanSwift/CleanSwift/Astronomy/AstronomyInteractor.swift

```diff
@@ -0,0 +1,55 @@
+// AstronomyInteractor.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Интерактор экрана Астрономии
+final class AstronomyInteractor: AstronomyBusinessLogic, AstronomyDataStoreProtocol {
+    // MARK: - Public Properties
+
+    var title: String?
+    var explanation: String?
+
+    // MARK: - Private Properties
+
+    private var worker: AstronomyNetworkWorkerProtocol?
+    private var presenter: AstronomyPresentationLogic?
+
+    // MARK: - Initializers
+
+    init(worker: AstronomyNetworkWorkerProtocol?, presenter: AstronomyPresentationLogic) {
+        self.worker = worker
+        self.presenter = presenter
+    }
+
+    // MARK: - Public Methods
+
+    func requestInitForm(_ request: AstronomyModel.InitForm.Request) {
+        worker?.fetchPictureOfDay { [weak self] result in
+            guard let self else { return }
+            switch result {
+            case let .success(picture):
+                self.fetchImage(imagePath: picture.imagePath)
+                self.explanation = picture.explanation
+                let response = AstronomyModel.InitForm.Response(pictureOfDay: picture)
+                self.presenter?.presentInitForm(response: response)
+            case let .failure(error):
+                print(error.localizedDescription)
```

Comment:
```suggestion
                print(error.localizedDescription)
```
принты вроде нужно заменить на алерты